scripts/http-cat.coffee

```diff
@@ -15,7 +15,7 @@
 #   bwillis
 
 module.exports = (robot) ->
-  robot.hear /(\b[0-9]{3}\b)/i, (msg) ->
+  robot.respond /(\b[0-9]{3}\b)/i, (msg) ->
```

Comment:
This one can't even post anymore because it of the `return unless msg.message.room is "dev_interns"` line below. This is just blindly cutting of brehm's limbs :(